tools/ctl/command/runbenchmark/runbenchmark.go

```diff
@@ -102,6 +108,70 @@ func parseArgs(cfg *Config, args []string) error {
 	return nil
 }
 
+// compileToFS builds a Go program and loads the binary directly into the provided memfs.
+func compileToFS(ctx context.Context, fs billy.Filesystem, srcPath string) error {
+	if filepath.Base(srcPath) == "main.go" {
```

Comment:
what's up here? this is a valid (and sometimes the only) way of building a specific package. i think we're safe to just drop this